repeatmasking_utils.py

```diff
@@ -0,0 +1,663 @@
+# See the NOTICE file distributed with this work for additional information #pylint: disable=missing-module-docstring
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import logging
+import multiprocessing
+import os
+import pathlib
+import re
+import subprocess
+
+from pathlib import Path
+
+from typing import Union
+from utils import (
+    create_dir,
+    check_exe,
+    check_gtf_content,
+    get_seq_region_lengths,
+    create_slice_ids,
+    slice_output_to_gtf,
+    get_sequence,
+)
+
+logger = logging.getLogger(__name__)
+
+
+def run_repeatmasker_regions(  # pylint: disable=too-many-arguments
```

Comment:
a config JSON has been added to set the path and all the parameters related to the softwares. I think that the arguments of the command could be related just to the options the user would like to run and this kind of configuration might be done in the config file but let me know if you have a better idea

Reply:
However I would leave the parameters library and species as arguments of the command because it is easier adding them in the command from the pipeline

Reply:
Fine with me, probably worth having a ticket for a config if it's not already the case